features/step_definitions/checks_for_rails_security_issues_steps.rb

```diff
@@ -5,7 +5,7 @@ def executing_rails_security
 def rails_security_output
   `cd tmp/aruba/#{@projectname}/ && brakeman -q -p .`
     .split("\n")
-    .reject { |line| !line.match(/^\+|\|/) }
+    .reject { |line| !Warder::RailsSecurityRunner::PRINTABLE_REGEXP.match(line) }
```

Comment:
Line is too long. [81/80]<br>Place the . on the previous line, together with the method call receiver.